README.md

```
@@ -1,93 +1,190 @@
 
-<div align="center">
-  
-![Trophy](https://github-profile-trophy.vercel.app/?username=ThomasJButler&theme=matrix&no-frame=true&row=1&column=6)
 
 </div>
 
+<!-- Table of Contents -->
+- [About Me](#about-me)
+- [My Tech Stacks](#my-tech-stacks)
+- [Credentials & Specialisations](#credentials--specialisations)
+- [Focus Areas](#focus-areas)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)